File: taeyang/Python/Sujong/x_min.py
import sys

# n,x -> 1 ~ 1만
# x 보다 작은 수만 출력한다.

n, x = map(int, sys.stdin.readline().rstrip().split(" "))

a = list(map(int,sys.stdin.readline().rstrip().split(" ")))

answer = list()

for i in a:
    if i < x:
        answer.append(str(i))

print(" ".join(answer))

File: taeyang/Sujong/XMin.java
package Sujong;

import java.io.*;
import java.util.*;

/*

 */
public class XMin {

    public static void main(String[] args) throws IOException {
//        선언
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine(), " ");
//        초기화
        int n = Integer.parseInt(st.nextToken());
        int x = Integer.parseInt(st.nextToken());
        int[] arr = new int[n];
        String[] num = br.readLine().split(" ");
//        로직
        for (int i = 0; i < n; i++) {
            arr[i] = Integer.parseInt(num[i]);
        }

        List<String> list = new ArrayList<>();

        for (int i = 0; i < n; i++) {
            if (arr[i] < x) {
                list.add(String.valueOf(arr[i]));
            }
        }
//        출력
        System.out.println(String.join(" ", list));
    }
}
